app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminsController < ApplicationController
```

Comment:
Admin::DashboardController < Admin::BaseController I think will be the first line